### PR TITLE
Version updated and python 3.7 support added

### PR DESCRIPTION
**Pull request by sab**
Checklist
* [Х] Used a fork of the feedstock to propose changes
* [Х] Reset the build number to `0` (if the version changed)
* [Х] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [Х] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
   number: 0
   #  Yandex supplies whl files on PyPI for:
-  # - Linux: Python 2.7, 3.4, 3.5 and 3.6
-  # - OS X: Python 2.7, 3.4, 3.5 and 3.6
-  # - Win-64: Python 2.7, 3.4, 3.5 and 3.6
+  # - Linux: Python 2.7, 3.4, 3.5, 3.6 and 3.7
+  # - OS X: Python 2.7, 3.4, 3.5, 3.6 and 3.7
+  # - Win-64: Python 2.7, 3.4, 3.5, 3.6 and 3.7
   skip: True  # [win32]
   skip: True  # [py26]
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: We don't build py26 or py33

File: recipe/meta.yaml [hunk 2]
```diff
@@ -20,6 +20,7 @@ requirements:
     - python
     - numpy
     - six
+    - enum34
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: I think this may only be needed for py27

File: recipe/meta.yaml (hunk 1 above)
Comment (sab) [reply recipe/meta.yaml]: Thanks! Removed.

File: recipe/meta.yaml (hunk 2 above)
Comment (sab) [reply recipe/meta.yaml]: Thanks! Comment added.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
-  number: 0
+  number: 1
   #  Yandex supplies whl files on PyPI for:
-  # - Linux: Python 2.7, 3.4, 3.5 and 3.6
-  # - OS X: Python 2.7, 3.4, 3.5 and 3.6
-  # - Win-64: Python 2.7, 3.4, 3.5 and 3.6
+  # - Linux: Python 2.7, 3.4, 3.5, 3.6
+  # - OS X: Python 2.7, 3.4, 3.5, 3.6
+  # - Win-64: Python 2.7, 3.4, 3.5, 3.6
   skip: True  # [win32]
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: FYI we also don't build on win32 any more.

Comment (sab) [reply recipe/meta.yaml]: Thank you, Christopher. We'll remove this line next version update.

**Pull request merged**